pkg/test/resourcefixture/testdata/basic/compute/v1beta1/computeforwardingrule/privateserviceconnectforwardingrule/dependencies.yaml

```diff
@@ -0,0 +1,128 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#apiVersion: resourcemanager.cnrm.cloud.google.com/v1beta1
```

Comment:
Remove?

Reply:
I thought I reverted it after local testing. We want to use this dependency project for service attachment.